X/XRootD/build_tarballs.jl

```diff
@@ -3,68 +3,71 @@
 using BinaryBuilder, Pkg
 
 name = "XRootD"
-version = v"5.5.4"
+version = v"5.6.9"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/xrootd/xrootd/releases/download/v$(version)/xrootd-$(version).tar.gz", "41a8557ea2d118b1950282b17abea9230b252aa5ee1a5959173e2534b7d611d3")
+    ArchiveSource("https://github.com/xrootd/xrootd/releases/download/v$(version)/xrootd-$(version).tar.gz", "44196167fbcf030d113e3749dfdecab934c43ec15e38e77481e29aac191ca3a8")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
 mkdir build && cd build
 install_license ../xrootd-*/LICENSE
-cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DENABLE_PERL=FALSE ../xrootd-*/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_CXX_STANDARD=17 \
```

Comment:
If you want C+=17, you should really be requesting a newer GCC version. Most projects use GCC 7 with C++17.

Reply:
Yes, you are right. In other projects I used v8